poc/vdaf_prio3.py

```diff
@@ -483,6 +511,41 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+# Experimental multiproof variant of Prio3SumVec
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(valid_cls,
+                       num_proofs: Unsigned,
+                       field: FftField) -> bool:
```

Comment:
Avoid shadowing the imported `field` module
```suggestion
                       f: FftField) -> bool:
```